yaml/roads.yaml

```diff
@@ -353,6 +353,36 @@ filters:
       <<: *osm_footway_properties
       kind: path
       kind_detail: pedestrian
+  - filter:
+      highway: [track]
+      any:
+        - {surface: [paved, asphalt, concrete]}
```

Comment:
You don't need the `{}` around single items in a list context.

Reply:
Fixed in 11d447c59f029103f1a6fdc40125e4771b05519e.